cat/CaT-service.yaml

```diff
@@ -3877,6 +3877,10 @@ components:
               description: |+
                 Defines a the buyer can add new questions of this type
               type: boolean
+            order:
```

Comment:
Should we also add this to `QuestionGroup.nonOCDS`?  I think it was the group display that is highlighted in https://crowncommercialservice.atlassian.net/browse/SCAT-2783 🤔

Reply:
yes Agree, guess same for questions with in group. it need order field at group and question as well